source/blog/044-rfc-indented-syntax.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Request for Comments: Indented Syntax Improvements"
+author: James Stuckey Weber
+date: 2024-11-27 15:30:00 -8
+---
+
+For users of the indented syntax (`.sass`), there has been a long standing
+request for the ability to have line breaks that don't end a statement. Certain
+CSS syntaxes like `grid-template` and `@font-face` `src` declarations are long,
+and line breaks can greatly increase the readability of the styles. 
+
+There have been many suggestions, several attempts, and numerous workarounds to
+address this pain point. We are proposing several new syntax options in the
+indented syntax that add considerable flexibility and look to address this as
+part of the language.
+
+## The new syntax
+
+Instead of adding syntax that is completely new to Sass, we propose to borrow
+syntax from the SCSS format and make it available in the indented syntax. This
+reduces the learning curve, code complexity, and likelihood of future conflicts
+with new CSS features.
+
+A common way of using this syntax will be wrapping lists with linebreaks in
```

Comment:
Nit: `line break` instead of `linebreak` (also other instances below)

```suggestion
A common way of using this syntax will be wrapping lists with line breaks in
```